src/modules/missionHelper/i18n/de_DE.root.json

```diff
@@ -53,6 +53,10 @@
 			"description": "Blendet ELW 1 komplett aus und zeigt nur ELW 2 an.",
 			"title": "ELW 1 ausblenden"
 		},
+		"hoverTip": {
+				"description": "Tooltips für die Buttons wie der Overlay-Button",
```

Comment:
Are these indentations right?

Reply:
they are then set correctly during the build :)